### DISCUSSION TITLE
Amount of Firewood

### DISCUSSION THREAD

**Discussion by FeraelGare** [Royal-Modding/FS19_Firewood#8] [Q&A] [answered]
Hey i just notice that the amount of firewood witch i get is something odd. what i mean is, that i have a log which gives me 127 L of firewood but if i divide it with the chainesaw i get two pieces one with 80L and the other with 70 L . So i testet i bit further an notice that its better too cut as small pieces as possible before chopping it to Firewood. Tried it on different Maps with the same Result. Did i mess something up or is just as it is. shouldn´t the amount of Firewood be bount to the Mass of the log ?

**Comment by fcelsa** [comment, marked as answer]
It's a wanted behaviour, that push player to cut a small pieces for firewood like in real life; not remember all details without read the code, but in general the "rules" are: All pieces cut from first part of the tree, straight and length within 3 and 9 meters, are more convenient to sell at sawmill, branch and small cut made from the top of the tree are more profitable in firewood form, if cut into small enough pieces. 
(taking into account the limitations of the game) 
In all this, wood chips also play a role, when you make firewood from big pieces, you will obtain a yield similar or less than make woodchips processing, if you cut more, as you have noticed, you will have a higher yield.
I probably still missed some details, but it's also fun discover them in the game...
